### PR TITLE
Add Metrics

### PR DESCRIPTION
**Pull request by raulgomis**
This change adds the Metrics [awesome-metrics](https://github.com/raulgomis/awesome-metrics) list to this awesome list.

**https://github.com/raulgomis/awesome-metrics**

**A curated list of awesome metrics platforms & frameworks, ressources and other awesomeness.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title. For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.
- The list I added
	- **has been around for at least 30 days,**
	- is the result of hard work and is the best I could possibly produce,
	- is a non-generated Markdown file in a GitHub repo,
	- is not a duplicate,
	- includes a succinct description of the project/theme at the top ([example](https://github.com/willempienaar/awesome-quantified-self)),
	- includes a project logo/illustration (whenever possible) at the top right of the readme ([example](https://github.com/sindresorhus/awesome-electron)). (The logo should link to the project website or any relevant website. The logo should be high-DPI, so set it to maximum half the width of the original image.),
	- only has awesome items **with descriptions** *(Awesome lists are curations of the best, not of everything)*,
	- contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- has a Table of Contents section **named `Contents`** as the first section,
	- has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)* *(If you use a license badge, it should be SVG, not PNG)*,
	- has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- doesn't include a Travis badge,
	- has been checked for inconsistent formatting, spelling, and grammar:
		- Each link description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.

### CODE REVIEW

**Comment by davisonio** [comment]
> The entry in the Awesome list should include a short description about the list project/theme (not the list itself), and should be added at the bottom of the appropriate category.



> includes a succinct description of the project/theme at the top (example),

These are both not done.

Title H1 of the list could be `Awesome Metrics` instead of `awesome-metrics`. And I'd also remove the divider after the contents.

**Comment by raulgomis** [comment]
Thanks @davisonio ! Fixed!

**Comment by legodude17** [comment]
Some changes:
 * All of the heads should be in title case.
 * Check all of the descriptions to make sure they don't start with `A` or `An`.
 * Use an appropriate license like CC0, instead of MIT.

**Comment by raulgomis** [comment]
Thanks @legodude17 ! Fixed!

**Comment by legodude17** [comment]
You missed one:

> Kenshin - A time-series database alternative to Graphite Whisper with 40x improvement in IOPS.

**Comment by raulgomis** [comment]
@legodude17 thanks! fixed!

**Comment by legodude17** [comment]
@sindresorhus Any comments?

**Comment by sindresorhus** [comment]
There's a typo in the repo description `ressources`.

---

> includes a succinct description of the project/theme at the top (example),

:arrow_up:

---

The repo should have the following GitHub topics set: awesome-list, awesome, list. I encourage you to add more relevant topics.

---

I would like to see more than just libraries. Include some blog posts, tutorials, links to community resources (Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community), videos of talks, book, learning material, etc.

File: readme.md [hunk 1]
```diff
@@ -220,6 +220,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Serverless Framework](https://github.com/JustServerless/awesome-serverless)
 - [Apache Wicket](https://github.com/PhantomYdn/awesome-wicket) - Java web app framework.
 - [Vert.x](https://github.com/vert-x3/vertx-awesome) - Toolkit for building reactive apps on the JVM.
+- [Metrics](https://github.com/raulgomis/awesome-metrics) - Metrics platforms & frameworks for performance tuning & optimization.
```
Comment (sindresorhus) [inline readme.md]: > The entry in the Awesome list should include a short description about the list project/theme **(not the list itself)**

**Comment by raulgomis** [comment]
Thanks  @sindresorhus ! Fixed!

File: readme.md [hunk 2]
```diff
@@ -220,6 +220,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Serverless Framework](https://github.com/JustServerless/awesome-serverless)
 - [Apache Wicket](https://github.com/PhantomYdn/awesome-wicket) - Java web app framework.
 - [Vert.x](https://github.com/vert-x3/vertx-awesome) - Toolkit for building reactive apps on the JVM.
+- [Metrics](https://github.com/raulgomis/awesome-metrics) - Learn how to integrate a metrics platform in your solution for performance & tunning and other use cases.
```
Comment (sindresorhus) [inline readme.md]: The description is too long. Look at the other descriptions. Try to make it as succinct as possible.

**Comment by sindresorhus** [comment]
> includes a succinct description of the project/theme at the top (example),
> ⬆️

Not done.

> I would like to see more than just libraries. Include some blog posts, tutorials, links to community resources (Stack Overflow tag, IRC, chat, forums, communities, Reddit, etc. Example: https://github.com/sindresorhus/awesome-electron#community), videos of talks, book, learning material, etc.

Only partly done.

**Comment by sindresorhus** [comment]
ping

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**